src/zenml/utils/source_utils.py

```diff
@@ -457,11 +457,22 @@ def import_python_file(file_path: str) -> types.ModuleType:
         file_path: Path to python file that should be imported.
 
     Returns:
-        The imported module.
+        imported module: Module
     """
     # Add directory of python file to PYTHONPATH so we can import it
     file_path = os.path.abspath(file_path)
     sys.path.append(os.path.dirname(file_path))
```

Comment:
I am unsure whether manipulating the sys.path is a good practice here. It might lead to unexpected consequences. I do not have another suggestion at the moment though

Reply:
We also have the `source_utils.prepend_python_path(...)` contextmanager which only temporarily modifies the python path.